### PR TITLE
4.0.0 RC

### LINKED ISSUE #42: Not compatible with ampersand-form-view@3.x
Since upgrading to `v3.x` of `ampersand-form-view`, `ampersand-select-view` hasn't been rendering (either `v2.x` or `v3.x`)

[Here's a demo of it not working.](http://requirebin.com/?gist=f5d09550c1aaf6d89c63)

[Here's the same code – now working–, but with ampersand-form-view@2.x](http://requirebin.com/?gist=4a343694a1772363a4de)

### PR DESCRIPTION
**Pull request by cdaringe**
Closes #42, #2, #22 (tests added)

closing #39 will drop this commit diff down to just a few.  closing #39 is higher priority

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +60,8 @@ If using a collection to produce `<select>` `<option>`s, the following may also
 - `[textAttribute]`: model attribute to use as the text of the option node in the select box
 - `[yieldModel]`: [default: `true`] if options is a collection, yields the full model rather than just its `idAttribute` to `.value`
 
+When the collection changes, the view will try and maintain its currently `.value`.  If the corresponding model is removed, the <select> control will default to the 0th index <option> and update its value accordingly.
```
Comment (wraithgar) [inline README.md]: This appears to be breaking the readme formatting.

Comment (wraithgar) [inline README.md]: Ah nope it's already borked in master.

Comment (cdaringe) [inline README.md]: hmm what are you using as a previewer? my markdown previewer looks ok:

screen shot 2015-04-15 at 11 37 46 am

**Comment by wraithgar** [comment]
sync w/ master please?

File: README.md (hunk 1 above)
Comment (wraithgar) [inline README.md]: On this page itself it's even borked. https://github.com/cdaringe/ampersand-select-view/tree/4.0.0.  Fixed in master now though.

**Comment by cdaringe** [comment]
synced

**Comment by cdaringe** [comment]
additionally, reviewing with `?w=1` in the URL makes the diff look **much better** and easy to process

**Comment by wraithgar** [comment]
ok This looks ready to go, tests pass locally

**Comment by wraithgar** [comment]
Need another +1 from @AmpersandJS/core-team of course

**Comment by lukekarrys** [comment]
+1

**Pull request merged**